### PR TITLE
added amber golems to limak

### PR DESCRIPTION
**Pull request by bulian-bg**

### CODE REVIEW

File: ImprovedAnvil/scripts/encounters/random/S!limak.BAF [hunk 1]
```diff
@@ -83,6 +83,18 @@ THEN
...
 
+IF
+  Allegiance(Myself,ENEMY)
+  Detect([PC])
+  !HPLT(Myself,1)
+  Global("Limakamber","LOCALS",0)
+THEN
+  RESPONSE #100
+    SetGlobal("Limakamber","LOCALS",1)
+    CreateCreatureObjectEffect("S!amber","",Myself)  // Amber Golem
```
Comment (critto-bg) [inline ImprovedAnvil/scripts/encounters/random/S!limak.BAF:94]: You do not specify a visual effect here. How does the game visualize the appearance of the golems?

Comment (bulian-bg) [reply ImprovedAnvil/scripts/encounters/random/S!limak.BAF:94]: This seems to be the code present in the ultra golems script where the golems just appear.  Suggestions welcome.

from s!ultgoa.bcs
IF
	Allegiance(Myself,ENEMY)
	See([PC])
	!GlobalTimerNotExpired("golemcall","LOCALS")
	GlobalLT("golems","LOCALS",6)
	!HPLT(Myself,1)
THEN
	RESPONSE #100
		SetGlobalTimer("golemcall","LOCALS",FIVE_ROUNDS)
		IncrementGlobal("golems","LOCALS",1)
		CreateCreatureObjectEffect("S!amber2","",Myself)  // Amber Golem
END

Comment (critto-bg) [reply ImprovedAnvil/scripts/encounters/random/S!limak.BAF:94]: No, I was just making sure there won't be any weird stuff. Looks OK then, I'll merge it into the master.

**Comment by critto-bg** [comment]
I've merged and pushed this into my master. You can close this pull request and remove the respective branch.

**Pull request closed without merging**